pkg/agent/supportbundlecollection/support_bundle_controller.go

```diff
@@ -265,6 +266,35 @@ func (c *SupportBundleController) generateSupportBundle(supportBundle *cpv1b2.Su
 		return err
 	}
 
+	metricsDir := filepath.Join(basedir, "metrics")
+	if err = defaultFS.MkdirAll(metricsDir, 0755); err != nil {
```

Comment:
do we need to create a separate directory for metrics? It seems all other Dump* functions are writing files directly to baseDir.